components/alert/alert.css

```diff
@@ -0,0 +1,54 @@
+
+
+:root {
+  --border-color: rgb(0, 0, 0);
+  --danger-bg-color: #b80c00;
+  --warning-bg-color: rgb(233, 115, 18);
+  --success-bg-color: rgba(144, 221, 99, 0.671);
+  --danger-color: #F50F00;
+  --warning-color: rgb(230, 217, 198);
+  --success-color: rgb(31, 187, 44);
+}
+
+
+.alert-component {
+  border: 1px solid var(--border-color);
+  margin: 4rem;
+  padding: 2.5rem;
+  border-radius: 0.7rem;
+}
+
+.alert__btn {
+  font-size: 2rem;
```

Comment:
Why do you use underscore here, is there any reason? if not, use the same naming convention

Reply:
Ya I tried to use BEM convention...Will try keeping it consistent thanks.